### PR TITLE
feature: Add name and properties to Threat, and add Threat to Event definition.

### PR DESCRIPTION
**Pull request by josedonizetti**
Fix https://github.com/aquasecurity/tracee/issues/3741

Add name and properties to Threat, and add Threat to Event definition.

### CODE REVIEW

**Review by rafaeldtinoco — APPROVED** [review]
LGTM technically correct, feel free to merge or ask someone else for the +1 on naming and conventions if you think it is needed.

**Review by yanivagman — APPROVED** [review]
LGTM

File: api/v1beta1/definition.proto [hunk 1]
```diff
@@ -4,12 +4,15 @@ option go_package = "github.co/aquasecurity/tracee/api/v1beta1";
...
 
+import "api/v1beta1/threat.proto";
+
 message EventDefinition {
 	int32 id = 1;
 	string name = 2;
 	Version version= 3;
 	string description = 4 ;
 	repeated string tags = 5;
+	optional Threat threat = 6;
```
Comment (NDStrahilevitz) [inline api/v1beta1/definition.proto:15]: nit: I think that this should be called properties to align with the internal event definition concept, or vice versa, your call.

**Review by NDStrahilevitz — APPROVED** [review]

File: api/v1beta1/definition.proto (hunk 1 above)
Comment (josedonizetti) [reply api/v1beta1/definition.proto:15]: hmm, I think Threat is best, because we already have properties on it.

**Pull request merged**